Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/Action/MultiplyDialog.cs

```diff
@@ -0,0 +1,68 @@
+using System;
```

Comment:
Here the copyright header is missing.

Reply:
Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)